### PR TITLE
Update README.md, changed ROS2 to ROS 2

### PR DESCRIPTION
**Pull request by IrinaTerebiznik**
Changed  every "ROS2"  that is not a command nor a URL to "ROS 2", as it is properly written that way.

### CODE REVIEW

File: inorbit_republisher/README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ This mapping option transforms the entire ROS message to a JSON string.
 The `mapping_options` for this type include:
 
 * `mapping_type`: this mapping option should be set to `serialize`.
-* `fields`: (optional) a set of first level fields or keys to keep. If not provided, all fields are kept. For example, using the following mapping option for serializing 4 [NavSatFix](https://docs.ros2.org/latest/api/sensor_msgs/msg/NavSatFix.html) fields:
+* `fields`: (optional) a set of first level fields or keys to keep. If not provided, all fields are kept. For example, using the following mapping option for serializing 4 [NavSatFix](https://docs.ROS 2.org/latest/api/sensor_msgs/msg/NavSatFix.html) fields:
```
Comment (FlorGrosso) [inline inorbit_republisher/README.md]: @IrinaTerebiznik hi! Please note that here you will be breaking the URL with this change. It should still be `ros2`

File: inorbit_republisher/README.md [hunk 2]
```diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
...
-### Start ROS2 docker container (optional)
+### Start ROS 2 docker container (optional)
 
 You can run the commands below for building and running the republisher inside a docker container.
 
 ```bash
 docker run -ti --rm \
-  --workdir /root/ros2_ws/ \
-  -v .:/root/ros2_ws/src/inorbit_republisher \
+  --workdir /root/ROS 2_ws/ \
```
Comment (FlorGrosso) [inline inorbit_republisher/README.md]: Same here

File: inorbit_republisher/README.md [hunk 3]
```diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
...
+  -v .:/root/ROS 2_ws/src/inorbit_republisher \
```
Comment (FlorGrosso) [inline inorbit_republisher/README.md]: And here

File: inorbit_republisher/README.md [hunk 4]
```diff
@@ -197,7 +197,7 @@ colcon build --packages-select inorbit_republisher --symlink-install
 ```bash
 source install/local_setup.bash
 # Using the launch file under the 'launch' directory
-ros2 launch inorbit_republisher example.launch.xml
+ROS 2 launch inorbit_republisher example.launch.xml
```
Comment (FlorGrosso) [inline inorbit_republisher/README.md]:
```suggestion
ros2 launch inorbit_republisher example.launch.xml
```

File: inorbit_republisher/README.md [hunk 5]
```diff
@@ -173,21 +173,21 @@ These values will be published as latched and delivered only once every time a s
...
+  --workdir /root/ROS 2_ws/ \
+  -v .:/root/ROS 2_ws/src/inorbit_republisher \
   osrf/ros:foxy-desktop
 ```
 
 ### Build
 
 ```bash
-cd ~/ros2_ws
+cd ~/ROS 2_ws
```
Comment (FlorGrosso) [inline inorbit_republisher/README.md]:
```suggestion
cd ~/ros2_ws
```

**Comment by IrinaTerebiznik** [comment]
Second commit should fix those errors

**Review by FlorGrosso — APPROVED** [review]

**Pull request merged**